spec/javascripts/learning_questions/LearningQuestionFormCtrlSpec.js

```diff
@@ -0,0 +1,150 @@
+(function() {
+  'use strict';
+
+  describe('Controller: LearningQuestionForm', function() {
+    var controller,
+        LearningQuestionService,
+        $scope,
+        $q;
+
+    beforeEach(function() {
+      module('PDRClient');
+      inject(function(_$controller_, _$rootScope_, _$q_, $injector) {
+        $q = _$q_;
+        $scope = _$rootScope_.$new(true);
+        LearningQuestionService = $injector.get('LearningQuestionService');
+
+        $scope.learningQuestionForm = {};
+        $scope.context = 'testContext';
+
+        controller = _$controller_('LearningQuestionFormCtrl', {
+          $stateParams: {id: 1},
+          $scope: $scope,
+          LearningQuestionService: LearningQuestionService
+        });
+      });
+    });
+
+    describe('on initialization', function() {
+      it('invokes sets the context on the service', function() {
+        expect(LearningQuestionService.context).toEqual('testContext');
+      });
+    });
+
+    describe('#clearInputForm', function() {
+      var model = {body: 'Hello there'};
+
+      beforeEach(function() {
+        controller.clearInputForm(model);
+      });
+
+      it('blanks out the model body', function() {
+        expect(model.body).toEqual('');
+      });
+    });
+
+    describe('#validate', function() {
+      var data = 'This is some data';
+
+      beforeEach(function() {
+        spyOn(LearningQuestionService, 'validate');
+        controller.validate(data);
+      });
+
+      it('delegates to LearningQuestionService', function() {
```

Comment:
From the view of this controller, all it's doing is delegating to the service which does the heavy lifting.  If we change what it's delegating to, that is a valid test failure.

If the implementation changes, I would expect the `LearningQuestionService`'s tests to catch any breakage, since that service is doing the main body of work.